test/EnsembleKalmanProcessModule/runtests.jl

```diff
@@ -0,0 +1,219 @@
+using Distributions
+using LinearAlgebra
+using Random
+using Test
+
+using EnsembleKalmanProcesses.EnsembleKalmanProcessModule
+using EnsembleKalmanProcesses.ParameterDistributionStorage
+@testset "EnsembleKalmanProcessModule" begin
```

Comment:
These need to be updated to the new module notation.

Reply:
Actually, missed that `test/EnsembleKalmanProcessModule/runtests.jl` was renamed to `test/EnsembleKalmanProcess/runtests.jl` in `main` (and edited further). In c1b164d I implemented the changes I had made to the EKPModule tests in the EKP tests, and deleted the EKPModule tests. Verified that updated tests pass on 1.6 and 1.7.